src/__init__.py

```diff
@@ -90,3 +90,41 @@ def read_config(*items) -> list or object:
                 return_list.append(value)
 
     return return_list if len(items) > 1 else return_list[0]
+
+
+def set_config(path, value, action="replace", type_=str):
+    """Sets field to given value in configuration file.
```

Comment:
This docstring should definitely describe arguments and options; at least accepted values of action.